lms/views/basic_lti_launch.py

```diff
@@ -211,7 +205,6 @@ def unconfigured_basic_lti_launch(self):
         # Add the config needed by the JavaScript document selection code.
         self.context.js_config.config.update(
             {
-                "mode": "content-item-selection",
```

Comment:
This is now done by the `self.context.js_config.enable_content_item_selection_mode()` call above. This is the one view in `BasicLTILaunchViews` that needs `"content-item-selection"` mode instead of `"basic-lti-launch"` mode